Cargo.lock

```diff
@@ -1,21 +1,21 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 4
+version = 3
```

Comment:
hmm I think you may want to update your rust installation (personally I'm on rustc and cargo 1.83)